go.mod

```diff
@@ -17,7 +17,8 @@ require (
 	github.com/nats-io/nats-server/v2 v2.7.4 // indirect
 	github.com/nats-io/nats.go v1.13.1-0.20220308171302-2f2f6968e98d
 	github.com/nspcc-dev/hrw v1.0.9
-	github.com/nspcc-dev/neo-go v0.98.0
+	github.com/nspcc-dev/neo-go v0.98.3-pre.0.20220321144433-3b639f518ebb
+	github.com/nspcc-dev/neo-go/examples/nft-nd-nns v0.0.0-20220325151650-c039133acf68
```

Comment:
Why not neo-go v0.98.2 and why examples?

Reply:
fixed